.github/workflows/fork-ci-reusable.yml

```diff
@@ -0,0 +1,230 @@
+name: Reusable Fork Docker CI
+
+on:
+  workflow_call:
+    inputs:
+      registry-internal:
+        description: "Address of the internal image registry to push images to. For all releases except GA"
+        type: string
+        required: true
+      registry-public:
+        description: "Address of the public image registry to push images to. Only for GA releases"
```

Comment:
GA publishing will not be done through github actions.
Maybe we should just push to internal always.

Reply:
Let's drop the registry-public completely then